pandas/core/ops.py

```diff
@@ -900,6 +900,42 @@ def invalid_comparison(left, right, op):
     return res_values
 
 
+# -----------------------------------------------------------------------------
+# Dispatch logic
+
+def should_series_dispatch(left, right, op):
```

Comment:
There are a handful of dispatch functions scattered about this module.  In a follow-up I'd like to collect them in one place.